### ISSUE TITLE
[v3] Broken handle with shadow after upgrading to v3.1.0

### ISSUE THREAD

**Issue by chybisov** [gorhom/react-native-bottom-sheet#183]
# Bug

After upgrading to v3.1.0 handle with shadow now looks like this. I think it's due to changes in bottomSheet.tsx layout.

![image](https://user-images.githubusercontent.com/18644653/104194918-9def5700-542a-11eb-8bd9-3bca8077c5dd.png)

Styles:

```
contentContainer: {
    flex: 1,
    alignItems: 'center',
    backgroundColor: 'white',
    paddingVertical: 16,
  },
  bottomSheetBackgroundContainer: {
    backgroundColor: 'white',
    borderTopLeftRadius: 16,
    borderTopRightRadius: 16,
  },
  bottomSheetHandleContainer: {
    alignContent: 'center',
    alignItems: 'center',
    justifyContent: 'center',
    backgroundColor: 'white',
    paddingVertical: 12,
    borderTopLeftRadius: 16,
    borderTopRightRadius: 16,
    shadowColor: 'black',
    shadowOffset: {
      width: 0,
      height: 0,
    },
    shadowOpacity: 0.1,
    shadowRadius: 4,
    elevation: 16,
  },
```

Components

```
...
      <BottomSheet
        ref={bottomSheetRef}
        index={0}
        snapPoints={SNAP_POINTS}
        onChange={handleSheetChanges}
        enableOverDrag={false}
        enableFlashScrollableIndicatorOnExpand={false}
        animateOnMount
        enableContentPanningGesture={false}
        handleComponent={BottomSheetHandle}
        backgroundComponent={BottomSheetBackground}
      >
        <View style={styles.contentContainer}>
          <Text>Test 🎉 </Text>
        </View>
      </BottomSheet>
...

const BottomSheetHandle = () => (
  <View style={styles.bottomSheetHandleContainer}>
    <View style={styles.indicator} />
  </View>
);

const BottomSheetBackground = ({ pointerEvents, style }: BottomSheetBackgroundProps) => (
  <View pointerEvents={pointerEvents} style={[style, styles.bottomSheetBackgroundContainer]} />
);
```

## Environment info

| Library                         | Version |
| ------------------------------- | ------- |
| @gorhom/bottom-sheet         | 3.1.0   |
| react-native                    | 0.63.4   |
| react-native-reanimated         | 2.0.0-rc.0   |
| react-native-gesture-handler    | 1.9.0   |

**Comment by chybisov** [comment]
@gorhom hi :) in example above if we move elevation and shadow styles from Handle to Background component then shadow appear as expected on iOS, but on Android all other content appears under Background component. It seems elevation has weird behavior with position: 'absolute' prop.

In light of the above what's the right way to do bottom sheet shadow? In Background or in Handle component? :)

**Comment by gorhom** [comment]
yeah ! i am aware of that , going to prepare a new way of applying style to the sheet container it self to resolve this issue

**Comment by chybisov** [comment]
@gorhom thank you! Hope to see this soon :)

**Comment by gorhom** [comment]
this been released with 3.2.0 🎉
